models/DamagedHelmet/glTF/DamagedHelmet.gltf

```diff
@@ -1,204 +1,235 @@
 {
-    "accessors" : [
+    "asset": {
+        "generator": "Maya2glTF",
```

Comment:
The DamagedHelmet model here comes from the sample model repository, and prior to that it was downloaded from SketchFab back before glTF 2.0 was even published.  It does need fixing, and possibly it should be simply re-exported to glTF using SketchFab's current glTF exporter.

But in any case, I don't want this copy of DamagedHelmet to be different from the one on glTF-Sample-Models, regardless of where we get the fixes from.

Reply:
My bad, it seems I forgot to make a patch-request branch. This new helmet export was never intended to be part of this patch, mea culpa

Reply:
Ah I see, this is off your `master` branch.  You may want to close this whole PR and open a new one from a patch branch.

Reply:
Will do so now.